src/components/cases/case.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { fetchedCases } from '../../Redux/Cases/cases';
+import './case.css';
+import icon from '../../assets/back-icon.jpeg';
+import SearchBar from '../search/search';
+
+export default function Home() {
+  const dispatch = useDispatch();
+  const allCases = useSelector((state) => state.covid);
+  const [search, setSearch] = useState('');
+  const cases = allCases.filter((item) => item.country_name
+    .toLowerCase().includes(search.toLowerCase()));
+
+  const handleOnChange = (e) => {
+    setSearch(e.target.value);
+  };
+  useEffect(() => {
+    dispatch(fetchedCases());
+  }, []);
+  return (
+    <div className="all-countries">
+      <div><SearchBar handleOnChange={handleOnChange} value={search} /></div>
+      <div className="country-container">
+        {cases.map((elt) => (
+          <ul key={elt.country_name} className="country">
+            <li>
+              <Link to={`/Info/${elt.country_name}`}>
+                <img
+                  className="back-img"
+                  src={icon}
+                  alt="back-icon"
+                  width={30}
+                  height={30}
+                />
+              </Link>
+              <div className="country-image">
+                <img src={elt.country_flag} alt="country-flag" />
+              </div>
+              <h2 className="country-name">{elt.country_name}</h2>
+              <p className="total-cases">
+                Total cases :
+                {elt.country_cases}
+              </p>
+            </li>
+          </ul>
+        ))}
+      </div>
+    </div>
```

Comment:
- [ ] Good job making the app responsive as requested, however, you were also requested to follow the design. Kindly implement the **alternating colors** as we have in the design.

  From the image below you can see a pattern - the first and fourth countries have lighter background colors, while the second and third have thicker backgrounds, and so on, whereas in your design all countries on the left have same background.

  Your Implementation | Given Design
  :---:|:---:
  ![girl-1](https://user-images.githubusercontent.com/54848363/175662056-167a4524-e3ce-481f-9a2d-8dc175540446.jpg) | ![girl-2](https://user-images.githubusercontent.com/54848363/175662048-ec065bbf-68d1-416b-98d7-eb9d0285d1fd.jpg)